core/src/main/java/org/apache/iceberg/SerializableTable.java

```diff
@@ -361,6 +362,19 @@ protected Table newTable(TableOperations ops, String tableName) {
     }
   }
 
+  private static class SerializableConfSupplier implements SerializableSupplier<Configuration> {
```

Comment:
maybe we can merge the nested `SerializableConfiguration` class code into this supplier class?

Reply:
Yes, that make sense !